CMakeLists.txt

```diff
@@ -15,6 +15,8 @@ option(REGISTER_POKE_KERNELS "Use the faster way of register to run kernels" ON)
 option(BUILD_DEBUG "Build with debugging symbols. Otherwise build for performance" OFF)
 # Option to enable/disable cross compilation
 option(CROSS_COMPILE "Cross compile for Raspbian" OFF)
+# Option whether to create deb package
```

Comment:
Can you re-base your PR, so this file is not changed twice? If that is too much trouble, I could pull this version, github seems to be able to resolve duplicate identical changes.